web/components/commons/view_mixins.py

```diff
@@ -33,7 +33,7 @@ def run_pipeline(self, with_permalinks=False):
         except Exception as e:
             pipeline = None
             data['report']['error_title'] = e.__class__.__name__
-            data['report']['error_message'] = e.msg
+            data['report']['error_message'] = str(e)
```

Comment:
The reason I made this change was that some exceptions don't have a `msg` attribute, and this causes another exception (in my case it was `TypeError`). I'll use the `compat.str` here.